### ISSUE TITLE
Multiple grammars in the same file doesn't compile

### ISSUE THREAD

**Issue by balacij** [yegor256/naive-ebnf#34]
First off, thank you for this package -- simple syntax, beautiful results.

I encountered a small issue with trying to use `ebnf` environments in the same TeX file. It seems that the `ebnf` environment defines some commands for each use, so if we use the `ebnf` environment a second time, those 'new' definitions cause issues for latex compilers ('already defined' errors).

For example, pulling on an example from the README, if you have the following TeX:
```latex
\documentclass{article}
\usepackage{naive-ebnf}
\begin{document}
\begin{ebnf}
<Expression> := <Variable>
  | "$\lambda$" <Variable> "." <Expression>
  | "(" <Expression> <Expression> ")"
\end{ebnf}

Text.

\begin{ebnf}
<Expression> := <Variable>
  | "$\lambda$" <Variable> "." <Expression>
  | "(" <Expression> <Expression> ")"
\end{ebnf}
\end{document}
```

And you compile it with `latexmk example.tex -pdf`, you will find the following error:

```
! LaTeX Error: Control sequence \ebnf_curled already defined.

For immediate help type H <return>.
 ...                                              
                                                  
l.16 \end{ebnf}
```

As a workaround, I imported `expl3` and added the following TeX code after each of the ebnf environment:

```latex
\ExplSyntaxOn
\cs_if_exist:NT \ebnf_curled { \cs_undefine:N \ebnf_curled }
\cs_if_exist:NT \ebnf_brackets { \cs_undefine:N \ebnf_brackets }
\cs_if_exist:NT \ebnf_squares { \cs_undefine:N \ebnf_squares }
\ExplSyntaxOff
```

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.17`

**Comment by yegor256** [comment]
@balacij I'm sorry for missing your bug report! Now, I believe, it's fixed (in 536cc61) and will be released to CTAN within 24 hours.

**Comment by balacij** [comment]
No worries! Thank you for fixing this, @yegor256 !
